### ISSUE TITLE
All languages in all breadcrumbs

### ISSUE THREAD

**Issue by Ainali** [standard-for-public-code/community-translations-standard#62]
With multiple languages, our breadcrumbs solution picks up all languages. For instance, see the top on [this criteria](https://translations.standardforpubliccode.org/da/criteria/document-objectives.html) where Spanish, Danish and Chinese are all on previous and next.

**Comment by clausmullie** [comment]
Yup, fix is being worked on [here](https://github.com/standard-for-public-code/jekyll-theme/pull/10)
